### PR TITLE
BREAKING: O3-2424: Rename test-results-app to be labs-app

### PR DESCRIPTION
**Pull request by brandones**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] My work includes tests or is validated by existing tests.

## Summary
Just moving files around. Renames esm-patient-test-results-app to esm-patient-labs-app. Gives some files the `.extension.tsx` extension.

## Related Issue
https://issues.openmrs.org/browse/O3-2424

## Other
Corresponding change in the distro: https://github.com/openmrs/openmrs-distro-referenceapplication/pull/745

### CODE REVIEW

**Comment by brandones** [comment]
Would greatly appreciate a quick review since this will be very annoying to merge with any other work that comes up.

File: packages/esm-patient-labs-app/README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
-# esm-patient-test-results-app
+# esm-patient-labs-app
 
-The test results widget. It provides tabular and chart-based overviews of the test results available for a patient. 
+Funcitonality for viewing test results and ordering labs.
```
Comment (denniskigen) [inline packages/esm-patient-labs-app/README.md:3]:
```suggestion
Functionality for viewing test results and ordering labs.
```

**Review by denniskigen — APPROVED** [review]
Thanks, @brandones.

**Review by ibacher — APPROVED** [review]

**Comment by ibacher** [comment]
LGTM. I've marked this as breaking not because we need a major version, but because all down-stream users of the RefApp are likely to need to update their module lists to account for this (as will we).

**Pull request merged**